git-work.sh

```diff
@@ -46,12 +46,19 @@ case "$1" in
   # commit current work
   commit)
     [ -z "$2" ] && ( usage && exit 1 )
-    branch=$(git symbolic-ref --short HEAD)
-    if [[ $branch =~ ^-?[0-9]+$ ]]; then
-      id="#$branch: "
-    fi
     git add .
-    git commit -am "$id$2"
+    # checks is has an id
+    if [[ "$2" =~ ^\#-?[0-9]+ ]]; then
```

Comment:
@fabriciofx do the script need these two checks? I believe we can check only once as line 56.  What do you think?